### PR TITLE
@W-7930419@ Add html output option

### PR DESCRIPTION
**Pull request by jbartolotta-sfdc**
Added a simple html template that uses datatables.net to display the results.

Consolidate test validation that is the same

### CODE REVIEW

File: html-templates/simple.mustache [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
```
Comment (jbartolotta-sfdc) [inline html-templates/simple.mustache:1]: Simple template to output the data in a table. I will file 3pps for all of the new libs.

File: package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
 		"eslint": "^6.8.0",
 		"find-java-home": "^1.1.0",
 		"globby": "^11.0.0",
+		"mustache": "^4.0.1",
```
Comment (jbartolotta-sfdc) [inline package.json:18]: for templating.

File: src/commands/scanner/run.ts [hunk 3]
```diff
@@ -87,7 +87,7 @@ export default class Run extends ScannerCommand {
 			char: 'f',
 			description: messages.getMessage('flags.formatDescription'),
 			longDescription: messages.getMessage('flags.formatDescriptionLong'),
```
Comment (jbartolotta-sfdc) [inline src/commands/scanner/run.ts:89]: I reordered the options to be in alphabetical order, done in a couple of places.

File: src/commands/scanner/run.ts [hunk 4]
```diff
@@ -203,12 +203,11 @@ export default class Run extends ScannerCommand {
 		} else {
 			const fileExtension = outfile.slice(lastPeriod + 1);
 			switch (fileExtension) {
-				case OUTPUT_FORMAT.JSON:
-					return OUTPUT_FORMAT.JSON;
 				case OUTPUT_FORMAT.CSV:
```
Comment (jbartolotta-sfdc) [inline src/commands/scanner/run.ts:206]: Simplified some branching logic.

File: test/commands/scanner/run.test.ts [hunk 5]
```diff
@@ -33,6 +33,21 @@ describe('scanner:run', function () {
 
 	describe('E2E', () => {
 		describe('Output Type: XML', () => {
+			function validateXmlOutput(xml: string): void {
```
Comment (jbartolotta-sfdc) [inline test/commands/scanner/run.test.ts:36]: Consolidated validation logic that was duplicated.

File: test/commands/scanner/run.test.ts [hunk 6]
```diff
@@ -342,7 +311,113 @@ describe('scanner:run', function () {
 				});
 		});
 
+		describe('Output Type: HTML', () => {
```
Comment (jbartolotta-sfdc) [inline test/commands/scanner/run.test.ts:314]: New HTML tests

File: package.json (hunk 2 above)
Comment (rmohan20) [reply package.json:18]: nice find!

**Comment by jbartolotta-sfdc** [comment]
> This looks good to me, but I'm curious if we have any intention of allowing the user to supply their own template eventually.

I think that it could be useful at some point.

**Pull request merged**